src/Cli.Tests/EndToEndTests.cs

```diff
@@ -326,6 +326,36 @@ public void TestUpdateDepthLimitInGraphQLRuntimeSettings(string depthLimit, bool
         Assert.AreEqual(isSuccess, isError == 0);
     }
 
+    /// <summary>
+    /// This test checks behavior of executing `dab configure --runtime.graphql.{key} value
+    /// </summary>
+    [DataTestMethod]
+    [DataRow("/updatedPath", true, DisplayName = "Success in updating graphQL path values.")]
+    [DataRow("/updated-Path", true, DisplayName = "Success in updating graphQL path values.")]
+    [DataRow("/updated_Path", true, DisplayName = "Success in updating graphQL path values.")]
+    [DataRow("updatedPath", false, DisplayName = "Failure due to '/' missing.")]
+    [DataRow("/updated Path", false, DisplayName = "Failure due to white spaces.")]
+    [DataRow("/updated.Path", false, DisplayName = "Failure due to reserved char '.'.")]
+    [DataRow("/updated@Path", false, DisplayName = "Failure due reserved chars '@'.")]
+    [DataRow("/updated/Path", false, DisplayName = "Failure due reserved chars '/'.")]
+    public void TestUpdateGraphQLPathRuntimeSettings(string path, bool isSuccess)
+    {
+        // Initialize the config file.
+        string[] initArgs = { "init", "-c", TEST_RUNTIME_CONFIG_FILE, "--host-mode", "development", "--database-type",
+            "mssql", "--connection-string", TEST_ENV_CONN_STRING };
+        Program.Execute(initArgs, _cliLogger!, _fileSystem!, _runtimeConfigLoader!);
+
+        Assert.IsTrue(_runtimeConfigLoader!.TryLoadConfig(TEST_RUNTIME_CONFIG_FILE, out RuntimeConfig? runtimeConfig));
+        Assert.IsNotNull(runtimeConfig);
+
+        // Act: Update the Path in the config file.
+        string[] runtimeArgs = { "configure", "-c", TEST_RUNTIME_CONFIG_FILE, "--runtime.graphql.path", path };
```

Comment:
can you add a test where you update all the values of graphql, ie. path, enabled, allow-introspection all in single command.